larq_compute_engine/tflite/tests/interpreter_test.py

```diff
@@ -49,15 +49,18 @@ def test_interpreter_multi_input(use_iterator):
     interpreter = Interpreter(converter.convert(), num_threads=2)
     assert interpreter.input_types == [np.float32, np.float32]
     assert interpreter.output_types == [np.float32, np.float32]
-    assert interpreter.input_shapes == [(1, 24, 24, 2), (1, 24, 24, 1)]
-    assert interpreter.output_shapes == [(1, 24 * 24 * 2), (1, 24 * 24 * 1)]
+    assert interpreter.input_shapes == [(1, 24, 24, 1), (1, 24, 24, 2)]
+    assert sorted(interpreter.output_shapes) == [(1, 24 * 24 * 1), (1, 24 * 24 * 2)]
```

Comment:
The input order seems to be reversed and the output order was random in my case. Not sure if that is an issue, but I made the test such that it works on my machine.